### DISCUSSION TITLE
Postfix timeout & Mailclient login not possible

### DISCUSSION THREAD

**Discussion by designpixel** [modoboa/modoboa#2245] [Q&A]
Hi community,

I have searched through the forum but I didn't found an answer to my issue.
When I send mails on the server to different users, then all is fine and working. But I have two issues:
1. If I try to send a mail to an external mail address I got this error:

postfix/smtp[323783]: ... relay=none, delay=65644, delays=65584/0.07/60/0, dsn=4.4.1, status=deferred (connect to <external mailserver>:25 Connection timed out)

Does someone know where the root cause could be for this?

2. I'm not able to connect with Outlook on my Mac to this mail server
Error Message of the client: No Login possible.
I tried with SSL and without, I connected through the server like mail. xxx .net

I hope here is someone who likes to support me in finding the solution. Would be really nice!

Thanks & best regards
Christian

**Comment by tonioo** [comment]
@designpixel Do you have a firewall or something similar between your server and the Internet ?
